services/gateway/apis/http-version3/swagger/definitions/transactions.json

```
@@ -348,6 +348,85 @@
             }
         }
     },
+    "dryrunTransactionParamEnvelope": {
+        "type": "object",
+        "properties": {
+            "isSkipVerify": {
+                "type": "boolean",
+                "description": "A boolean indicator to skip transaction verification.",
+                "example": "false",
+                "default": false
+            },
+            "transaction": {
+                "type": "object",
+                "description": "Binary payload or the transaction object.",
+                "properties": {
+                    "module": {
+                        "type": "string",
+                        "example": "token",
+                        "description": "Blockchain module name related to the transaction."
+                    },
+                    "command": {
+                        "type": "string",
+                        "example": "transfer",
+                        "description": "Specific command name in the module to be executed."
+                    },
+                    "fee": {
+                        "type": "string",
+                        "example": "100000000",
+                        "description": "Transaction fee."
+                    },
+                    "nonce": {
+                        "type": "string",
+                        "example": "0"
+                    },
+                    "senderPublicKey": {
+                        "type": "string",
+                        "format": "senderPublicKey",
+                        "example": "a3f96c50d0446220ef2f98240898515cbba8155730679ca35326d98dcfb680f0",
+                        "description": "The public key is derived from the private key of the owner of the account.\nIt can be used to validate that the private key belongs to the owner, but not provide access to the owners private key.\n"
```

Could you please fix the same at all other places as well?
```suggestion
                        "description": "The public key is derived from the private key of the owner of the account.\nIt can be used to validate that the private key belongs to the owner, but not provide access to the owner's private key.\n"
```

---

Applied.